setup.py

```diff
@@ -26,7 +26,7 @@
         'cached_property>=1.0',
         'functools32;python_version<"3.2"',
         'h5py>=2.2',
-        'libsonata>=0.0.2',
+        'libsonata>=0.0.3',
```

Comment:
`uint64` dtype for synapse IDs comes from `libsonata>=0.0.3`; if we use previous `libsonata` versions, the recently fixed tests will be broken.